nncf/common/quantization/quantizer_propagation/solver.py

```diff
@@ -1155,6 +1155,11 @@ def _setup_initial_quantizers_for_operator_node(
                 pred_node_type
             ), "Invalid insertion point graph supplied for quantizer propagation!"
 
+            ip = pred_node[QuantizerPropagationStateGraph.QUANT_INSERTION_POINT_DATA_NODE_ATTR]
```

Comment:
Shouldn't we check that `QUANT_INSERTION_POINT_DATA_NODE_ATTR` exists first?

Reply:
The `assert` above serves the same purpose. It is not pretty, and I will have to refactor this when I get to refactoring the many main and auxiliary graph objects.